sql/parser/eval.go

```diff
@@ -471,6 +658,12 @@ var binOps = map[binArgs]func(Datum, Datum) (Datum, error){
 	binArgs{Div, floatType, floatType}: func(left Datum, right Datum) (Datum, error) {
 		return left.(DFloat) / right.(DFloat), nil
 	},
+	binArgs{Div, intervalType, intType}: func(left Datum, right Datum) (Datum, error) {
+		if right.(DInt) == 0 {
+			return DInterval{}, util.Error("division by zero error")
```

Comment:
Ben, should we return an error on those as well. Right now they would cause a server panic.